src/routes/index.ts

```diff
@@ -138,17 +138,28 @@ const DefaultRouteRequest = z.object({
 });
 
 const defaultRouteHandler: RequestHandler<{}, {}, z.infer<typeof DefaultRouteRequest>> = async (req, res, next) => {
-  const { step } = DefaultRouteRequest.parse(req.body);
-  if (!step) {
-    throw new Error('no step');
+  try {
+    const { step } = DefaultRouteRequest.parse(req.body);
```

Comment:
I'm kindof wondering if maybe the try/catch should only wrap this line?  Like below you're throwing in a couple of other cases that this new block will catch.  I'm not sure that's what you want to happen.

Reply:
With the step variable being used in the rest of the lines (except the await handler line), wouldn't they all have to be in the try block because of this? I was getting an error by only having the first line in try I think.  I can move await handler outside the try catch block